src/collection/collection.ts

```diff
@@ -157,16 +157,16 @@ export class Collection<T extends Document> {
     docs: InsertDocument<T> | InsertDocument<T>[],
     options?: InsertOptions,
   ) {
-    docs = Array.isArray(docs) ? docs : [docs];
```

Comment:
why did you make this change?

Reply:
typescript was complaining that the types were incompatible so had to declare a new variable